tree/ntupleutil/v7/src/RNTupleExporter.cxx

```diff
@@ -46,13 +46,27 @@ struct RColumnExportInfo {
    }
 };
 
-void AddColumnsFromField(std::vector<RColumnExportInfo> &vec, const RNTupleDescriptor &desc,
-                         const RFieldDescriptor &fieldDesc, const std::unordered_set<EColumnType> &filter,
-                         RNTupleExporter::EFilterType filterType)
+struct RAddColumnsResult {
+   int fNColsTotal;
+   int fNColsFilteredOut;
+
+   RAddColumnsResult &operator+=(const RAddColumnsResult &other)
+   {
+      fNColsTotal += other.fNColsTotal;
+      fNColsFilteredOut += other.fNColsFilteredOut;
+      return *this;
+   }
+};
+
+RAddColumnsResult AddColumnsFromField(std::vector<RColumnExportInfo> &vec, const RNTupleDescriptor &desc,
+                                      const RFieldDescriptor &fieldDesc, const std::unordered_set<EColumnType> &filter,
+                                      RNTupleExporter::EFilterType filterType)
 {
    R__LOG_DEBUG(1, RNTupleExporterLog()) << "processing field \"" << desc.GetQualifiedFieldName(fieldDesc.GetId())
                                          << "\"";
 
+   RAddColumnsResult res{};
```

Comment:
Took me a bit to realize that this actually zero-initializes the members (`RAddColumnsResult res;` would not!). Can we maybe add explicit member-initialization in `RAddColumnsResult`?